openslides_gui/gui.py

```diff
@@ -0,0 +1,755 @@
+            os.makedirs(dp)
+        with open(self.gui_settings_path, "w", encoding="utf-8") as f:
+            json.dump(settings, f, ensure_ascii=False, indent=4)
+
+    def apply_backup_settings(self):
+        if self.backupdb_enabled and self.server_running:
+            now = datetime.datetime.utcnow()
+            delta = datetime.timedelta(seconds=self.backup_interval_seconds)
+            ref = self.last_backup
+            if ref is None:
+                ref = now
+            ref += delta
+
+            d = ref - now
+            seconds = d.days * 86400 + d.seconds
+            if seconds < 1:
+                seconds = 30  # avoid backup immediatly after start
+            self.backup_timer.Start(seconds * 1000, True)
+        else:
+            self.backup_timer.Stop()
+
+    def do_backup(self):
+        cmd = [
+            sys.executable, "-u", "-m", "openslides", "backupdb",
+            self.backupdb_destination,
+        ]
+        p = subprocess.Popen(
+            cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT)
+        p.stdin.close()
+        output = p.stdout.read().strip()
+        exitcode = p.wait()
+        if output:
+            self.cmd_run_ctrl.append_message(output)
+
+        time = datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        if exitcode == 0:
+            self.cmd_run_ctrl.append_message(
+                _("{0}: Database backup successful.").format(time))
+        else:
+            self.cmd_run_ctrl.append_message(
+                _("{0}: Database backup failed!").format(time))
+
+        self.last_backup = datetime.datetime.utcnow()
+
+    def on_syncdb_clicked(self, evt):
+        self.cmd_run_ctrl.append_message(_("Syncing database..."))
+        self.cmd_run_ctrl.run_command("migrate")
+
+    def on_reset_admin_clicked(self, evt):
+        self.cmd_run_ctrl.append_message(_("Resetting admin user..."))
+        self.cmd_run_ctrl.run_command("createsuperuser")
+
+    def on_about_clicked(self, evt):
+        info = wx.adv.AboutDialogInfo()
+        info.SetName("OpenSlides")
+        info.SetVersion(openslides.__version__)
+        info.SetDescription(_(
+            "OpenSlides is a free web based presentation and "
+            "assembly system.\n"
+            "OpenSlides is free software; licensed under the MIT license."
+        ).replace(u" ", u"\u00a0"))
+        info.SetCopyright(_(u"\u00a9 2011-2015 by OpenSlides team"))
+        info.SetWebSite("http://www.openslides.org/", "www.openslides.org")
+
+        # XXX: at least on wxgtk this has no effect
+        info.SetIcon(self.GetIcon())
+        wx.adv.AboutBox(info)
+
+    def on_start_server_clicked(self, evt):
+        if self.server_running:
+            self.cmd_run_ctrl.cancel_command()
+            return
+
+        if self._host == "0.0.0.0":
+            args = ["--port", self._port]
+        else:
+            args = ["--address", self._host, "--port", self._port]
```

Comment:
OpenSlides master uses djangos default commands/parameters to start server with special port/address:
``runserver [port or address:port]`` (without any `--port` or `--address`)
see https://docs.djangoproject.com/en/1.8/ref/django-admin/#runserver-port-or-address-port
Can you please change and test it again? Thx.